### ISSUE TITLE
Show-stopping issues using two renderers

### ISSUE THREAD

**Issue by trakout** [GoodBoyDigital/pixi.js#1353]
If I initialise two renderers (using autoDetectRenderer) there are no immediate problems, until I start to add certain types of children to the second renderer, or set cacheAsBitMap as true to a display container in the second renderer, for example.

What's odd is that there seems to be no problem adding a graphics object to the second renderer, however I cannot add any other objects that I've loaded (eg. frames from the JSON loader).

Here is a bit of an example of the disconnect between renderers (http://jsfiddle.net/8o28nmzw/4/).. I create two renderers, add one displayObjectContainer for each, then add a graphic object to each container, then attempt to cache both displayObjectContainers. As you can see, console logs an error `[.WebGLRenderingContext]RENDER WARNING: texture bound to texture unit 0 is not renderable. It maybe non-power-of-2 and have incompatible texture filtering or is not 'texture complete'`.

This error comes up verbatim when I also try to add a new sprite on the second renderer (created from a previously used baseTexture for sprites in the first renderer). The error seems fairly generic to occur across these two operations, so I'm fairly clueless as to why it's happening.

Bottom line, either I'm doing something wrong, or there's a bug that's preventing me from using textures on two renderers at once, and also with certain other features like cacheAsBitmap.

**Comment by englercj** [comment]
v2 has a few issues with using shared resources across renderers and other abstraction leaks. Most of these issues have been solved in v3.

We will be ensuring that these use-cases are covered for sure before v3 is released.

**Comment by GoodBoyDigital** [comment]
Hi there! Just tested this in v3 and seems to work fine now. No more webGL errors :+1:  woop!
